content/en/tracing/guide/ingestion_sampling_use_cases.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Trace Sampling Use Cases
+kind: guide
+further_reading:
+- link: "/tracing/guide/trace_ingestion_volume_control/"
+  tag: "Guide"
+  text: "How to control ingested volumes"
+---
+
+## Overview
+
+Trace data tends to be repetitive. A problem in your application is rarely identified in only one trace and no others. For high throughput services, particularly for incidents that require your attention, an issue shows symptoms repeatedly in multiple traces. Consequently, there’s usually no need for you to collect every single trace for a service or endpoint, or every span within a trace. Datadog APM [ingestion control mechanisms][1] help you keep the visibility that you need to troubleshoot problems, while cutting down the noise and managing costs.
+
+Ingestion mechanisms are configurations within the Datadog Agent and Datadog tracing libraries. If you are using OpenTelemetry SDKs to instrument your applications, read [Ingestion Sampling with OpenTelemetry][2].
+
+This guide helps you understand when and how to use ingestion control configurations depending on the main use cases you might encounter. It covers:
+
+- [Determining which ingestion mechanisms are used](#determining-which-ingestion-mechanisms-are-used) for a given service
+- [Use cases that focus on keeping particular types of traces](#keeping-certain-types-of-traces)
+- [Use cases that focus on reducing ingested traces](#reducing-ingestion-for-high-volume-services)
+
+
+## Determining which ingestion mechanisms are used
+
+To identify which ingestion mechanisms are currently used in your Datadog environment, navigate to the [Ingestion Control Page][3].
+
+{{< img src="/tracing/guide/ingestion_sampling_use_cases/ingestion_control_page.png" alt="Ingestion Control Page" style="width:90%;" >}}
+
+The table gives insights on ingested volumes *by service*. The Configuration column provides a first indication of the current set up. It shows: 
+- `AUTOMATIC` if the sampling rate calculated in the Datadog Agent is applied to the traces that start from the service. Read more about the specifics of [Datadog Agent ingestion logic][5].
+- `CONFIGURED` if a custom sampling rate configured in the tracing library is applied to the traces that start from the service.
+
+Click on services to see details about what sampling decision makers (for example Agent or tracing library, rules or sample rates) are used for each service, as well as what [ingestion sampling mechanisms][1] are leveraged for ingested spans' services. 
+
+{{< img src="/tracing/guide/ingestion_sampling_use_cases/service-ingestion-summary.png" alt="Service Ingestion Summary" style="width:90%;" >}}
+
```

Comment:
What do you think about explaining this picture out more to really break out how to understand this view?

---

**Explanation of the Example**: 

In the Service Ingestion Summary example above, the **Ingestion reasons breakdown** table shows that most of the ingestion reasons for this service come from **rule** (user defined rule). 


The **Top sampling decision makers for this service** show that **web-store** service gets sampling decisions from **web-store**, **shopist-web-ui**, **shipping-worker**, **synthetics-browser**, and **product-recommendation**.

These five services all contribute in the overall sampling decisions that affect the **web-store** service. When determining how to fine tune the ingestion for **web-store**, all five services need to be considered.

Reply:
Good idea, I added that here !